network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -1,19 +1,31 @@
-<h2>{{product.name}}</h2>
+{% extends "./bg_base.html" %}
 
-<img src="{{mediaUrl}}{{product.image}}" width="500"/>
-<div>
-  {{product.company}}
+{% block body-id %}TODO-CATEGORY{% endblock %}
+
+{% block guts %}
+
+<div class="container">
+  <h2>{{product.name}}</h2>
+
+  <div class="creepometer mb-5"></div>
+
+  <img src="{{mediaUrl}}{{product.image}}" width="500"/>
+  <div>
```

Comment:
It feels like we've hit upon one of those few legitimate uses for `<table>` here.

Reply:
Possibly!